.github/workflows/ci.yaml

```diff
@@ -4,6 +4,13 @@ on:
     branches: [main]
   pull_request: null # target every PR
 jobs:
+  allchecks:
```

Comment:
#44 consolidates all the jobs into a single task `ci` (which runs `Earthly` for all of the tasks in one go).

Shall we name this nexus job `ci` in anticipation of that so we don't have to change branch protection again? (I guess it also matches the filename which is nice)